fuse-prometheus-operator.yml

```diff
@@ -0,0 +1,334 @@
+    scope: Namespaced
+    version: v1
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: prometheuses.monitoring.coreos.com
+    namespace: ${NAMESPACE}
+  spec:    
+    group: monitoring.coreos.com
+    names: 
+      kind: Prometheus
+      listKind: PrometheusList
+      plural: prometheuses
+      singular: prometheus
+    scope: Namespaced 
+    version: v1
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: alertmanagers.monitoring.coreos.com
+    namespace: ${NAMESPACE}
+  spec:
+    group: monitoring.coreos.com
+    names:
+      kind: Alertmanager
+      listKind: AlertmanagerList
+      plural: alertmanagers
+      singular: alertmanager
+    scope: Namespaced
+    version: v1
+
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: prometheus-operator
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+  rules:
+  - apiGroups:
+    - apiextensions.k8s.io
+    resources:
+    - customresourcedefinitions
+    verbs:
+    - '*'
+  - apiGroups:
+    - monitoring.coreos.com
+    resources:
+    - alertmanagers
+    - prometheuses
+    - prometheuses/finalizers
+    - alertmanagers/finalizers
+    - servicemonitors
+    - prometheusrules
+    verbs:
+    - '*'
+  - apiGroups:
+    - apps
+    resources:
+    - statefulsets
+    verbs:
+    - '*'
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    - secrets
+    verbs:
+    - '*'
+  - apiGroups:
+    - ""
+    resources:
+    - pods
+    verbs:
+    - list
+    - delete
+  - apiGroups:
+    - ""
+    resources:
+    - services
+    - endpoints
+    verbs:
+    - get
+    - create
+    - update
+  - apiGroups:
+    - ""
+    resources:
+    - nodes
+    verbs:
+    - list
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    verbs:
+    - list
+    - watch
+
+- apiVersion: apps/v1beta2
+  kind: Deployment
+  metadata:
+    labels:
+      k8s-app: prometheus-operator
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        k8s-app: prometheus-operator
+    template:
+      metadata:
+        labels:
+          k8s-app: prometheus-operator
+      spec:
+        containers:
+        - args:
+          - --manage-crds=false
+          - --namespace=${NAMESPACE}
+          - --logtostderr=true
+          - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
+          - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.23.2
+          image: quay.io/coreos/prometheus-operator:v0.23.2
```

Comment:
As a heads up, we just released v0.24.0.

Reply:
Updated to v0.24.0